Integrations/Cisco-umbrella/Cisco-umbrella.py

```diff
@@ -21,8 +21,7 @@
 
 API_TOKEN = demisto.params()['APIToken']
 BASE_URL = demisto.params()['baseURL']
-USE_SSL = demisto.params().get('insecure') if demisto.params().get('insecure') else \
-    not demisto.params().get('insecure_new')  # Backward compatibility issue, the old logic of insecure was reversed
```

Comment:
when did decide to remove this?
if we do, we have other integrations that should be changed.